tools/download-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+while [[ $# -gt 1 ]]
+do
+  key="$1"
+
+  case $key in
+      -t|--tap)
+      TAP="$2"
+      shift # past argument
+      ;;
+      *) # unknown option
+      ;;
+  esac
+  shift # past argument or value
+done
+
+if [ -z ${NODE_VERSION} ]; then
+  echo "NODE_VERSION is undefined! Please declare NODE_VERSION"
+  exit 1
+fi
+# Remove old files
+[ ! -z ${TAP} ]; rm -r ${TAP} || true
+#rm -r nodejs.org || true
+
+# Scrape the download server
+#wget -r --no-parent http://nodejs.org/dist/v${NODE_VERSION}/
```

Comment:
It gets further if I uncomment that but still fails later on:


iojs@nodejsb1:~/build/workspace/validate-downloads/temp$ sh build/tools/download-test.sh
build/tools/download-test.sh: 3: build/tools/download-test.sh: [[: not found
rm: missing operand
Try 'rm --help' for more information.
--2017-07-19 14:25:26--  http://nodejs.org/dist/v8.1.4/
Resolving nodejs.org (nodejs.org)... 104.20.22.46, 104.20.23.46, 2400:cb00:2048:1::6814:172e, ...
Connecting to nodejs.org (nodejs.org)|104.20.22.46|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
Saving to: 'nodejs.org/dist/v8.1.4/index.html'

    [ <=>                                                                                                 ] 5,355       --.-K/s   in 0s

2017-07-19 14:25:26 (14.5 MB/s) - 'nodejs.org/dist/v8.1.4/index.html' saved [5355]

Loading robots.txt; please ignore errors.
--2017-07-19 14:25:26--  http://nodejs.org/robots.txt
Reusing existing connection to nodejs.org:80.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://nodejs.org/robots.txt [following]
--2017-07-19 14:25:26--  https://nodejs.org/robots.txt
Connecting to nodejs.org (nodejs.org)|104.20.22.46|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/plain]
Saving to: 'nodejs.org/robots.txt'

    [ <=>                                                                                                 ] 112         --.-K/s   in 0s

2017-07-19 14:25:26 (30.8 MB/s) - 'nodejs.org/robots.txt' saved [112]

FINISHED --2017-07-19 14:25:26--
Total wall clock time: 0.05s
Downloaded: 2 files, 5.3K in 0s (14.7 MB/s)
cat: nodejs.org/dist/v8.1.4/SHASUMS256.txt: No such file or directory
build/tools/download-test.sh: 48: build/tools/download-test.sh: let: not found
build/tools/download-test.sh: 49: build/tools/download-test.sh: cannot create : Directory nonexistent
iojs@nodejsb1:~/build/workspace/validate-downloads/temp$